### ISSUE TITLE
[469d] Game won't run at first time after computer restart

### ISSUE THREAD

**Issue by garry-ut99** [OldUnreal/UnrealTournamentPatches#1380]
- game won't run at the first time after computer restart
- `UnrealTournament.exe` process is visible in TaskMgr (Ctrl + Alt + Del) 
- but nothing appears on the screen, no matter how long you wait
- after killing the process manually vis TaskMGr, and running the game again, it runs fine, and it always runs fine since then, until the next computer restart
- no log is procudced  (`UnrealTournament.log`), nor a dump crash, because it doesn't crash, only stucks infinitely
- (for me) happens on Win 7, but not on Win 10
- as far I remember it has been happening on 469c too

Not a critical problem, also Win 7 is deprecated, but who knows whether anybody is able to reproduce on Win 10/11 or Linux, or not, so still worth reporting, to make sure only Win 7 is afffected, also by the way maybe anybody else can try to help to found a solution, maybe there is some `-debug` switch to run the game in debug mode so it produces some log of what's going on or maybe there are some external tools which can help to diagnose stucked process.

**Comment by SeriousBuggie** [comment]
Open cmd.exe as administrator and run the following:
```
regsvr32 -u c:\windows\system32\gameux.dll
regsvr32 -u c:\windows\SysWOW64\gameux.dll
```
That should stop gameux from trying to intercept your games without you needing to delete the files

It is well known bug of Windows 7 and not related to patch.

You can found more info, if google by request `gameux win7 second run`.

**Comment by garry-ut99** [comment]
[Windows 7 Game failing to run](https://ut99.org/viewtopic.php?t=13392) - amusing, seems like another Win7's nasty issue, in addition to other issues like: [Windows Explorer - Green Ribbon of Death](https://www.google.com/search?q=win+7+green+ribbon+of+death) , [Desktop icons disappear](https://www.google.com/search?q=win7+desktop+icons+disappear) and [How to get rid of Internet Explorer in Win](https://www.techadvisor.com/article/727505/how-to-get-rid-of-internet-explorer-in-windows-7-and-windows-8.html)

Unregistering via `regsvr32 -u` hasn't worked (but I prefer to keep it unregistered, in the hope it helps to prevent any other related issues), but  registry fix / workaround `"Games"="127.0.0.1"` from the ut99 org link has worked (even with `gameux.dll` registered).  Thx, I forgot to check whether if it was a Windows problem.
